tools/reference-workspace/src/ldm/ext.ts

```diff
@@ -69,3 +71,9 @@ export const AmountWithRatio = modifySimpleMeasure(ReferenceLdm.Amount, (m) =>
 export const StageHistoryAttributeRef: ObjRef = {
     identifier: "attr.stagehistory.id",
 };
+/**
+ * Copy of ClosedYear attribute
+ */
+export const modifiedClosedYear: IAttribute = modifyAttribute(ReferenceLdm.ClosedYear, (m) =>
```

Comment:
Please use capital starting letter to stay consistent with the rest of the file.